README.md

```diff
@@ -49,6 +47,15 @@ The scripts will:
    that are required for the OpenShift installer - for example,
    creating the `default` libvirt storage pool and the `baremetal` and
    `provisioning` bridges.
+
+
+Once the aboave has been completed, additional scripts will preform the following:
```

Comment:
typo "preform" should be "perform"

Reply:
I removed this whole sentence because of the Note you recommended - it seemed redundant.